hibernate-search/api/src/main/java/org/wildfly/swarm/hibernate/search/HibernateSearchFraction.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.hibernate.search;
+
+import org.wildfly.swarm.spi.api.Fraction;
+
+/**
+ * @author George Gastaldi
+ */
+public class HibernateSearchFraction implements Fraction {
```

Comment:
I couldn't find any configuration options for this fraction (it doesn't add a subsystem to standalone.xml)